configuration-resources/large-language-models-llms/vertex-configuration.md

```diff
@@ -1,21 +1,29 @@
 # vertex-configuration
 
-A Google vertex configuration.
+A vertex-configuration resource is used for AI agents.
+
+{% hint style="warning" %}
+Note that the following examples sets sensitive data as plain text. This is for education purpose only, it's highly recommended to use [secrets](../../building-applications/secrets.md).
```

Comment:
do no write this, it is better to write good examples that people can copy/paste

Reply:
the problem is that you have to fill the secrets and it's better to let the user know what you need to put in the secrets

I can update the examples to use the secrets and fill the secrets with the values